source/common/formatter/substitution_formatter.cc

```diff
@@ -1174,6 +1187,85 @@ const StreamInfoFormatter::FieldExtractorLookupTbl& StreamInfoFormatter::getKnow
                 }
                 return result;
               });
+        }},
+       {"UDP_CLUSTER_NAME",
```

Comment:
I don't think you should have UDP specific versions of variables. I would keep them generic. Most of these can just use the generic versions. Then you can just update the documentation to discuss HTTP/TCP/UDP variants. Right now it just splits between HTTP and TCP. This is the page you will want to work on for documentation https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#command-operators. You will also need a release note in current.rst.

Reply:
> I don't think you should have UDP specific versions of variables. I would keep them generic. Most of these can just use the generic versions. Then you can just update the documentation to discuss HTTP/TCP/UDP variants. Right now it just splits between HTTP and TCP. This is the page you will want to work on for documentation https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#command-operators. You will also need a release note in current.rst.

OK, it looks more concise and easier to use and understand, thanks.